File: CSE221_Lab1/A_EvenOdd.py
n = int(input())
list = []
for i in range(n):
    list.append(int(input()))

for j in range(n):
    if list[j]%2 == 0:
        print(list[j] , " is an Even number.")
    else:
        print(list[j] , " is an Odd number.")        

File: CSE221_Lab1/A_EvenOdd.java
import java.util.*;

public class A_EvenOdd {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        ArrayList<Integer> list = new ArrayList<>();

        int input = sc.nextInt();

        for (int i = 0; i < input; i++) {
            list.add(sc.nextInt());
        }
        
        int num = 0;
        for (int j = 0; j < input; j++) {
            num = list.get(j);
            if (num%2 == 0) {
                System.out.println(num + " is an Even number.");
            } else {
                System.out.println(num + " is an Odd number.");
            }
        }
    }        
    
}